Products/CMFEditions/tests/test_IntegrationTests.py

```diff
@@ -24,6 +24,7 @@
 
 """
 from Acquisition import aq_base
+from plone.app.textfield.value import RichTextValue
```

Comment:
This should go to setup.py as well as a new test dependency, right?

Reply:
Ah yes, thanks.